src/tasks.py

```diff
@@ -68,8 +68,8 @@ def hello(c, name="world", verbose=0):
     _set_log_level(verbose)
     logger.debug("Set loglevel.")
 
-    print("Hello {}!".format(name))
-    logger.debug("Said hello.")
+    print("Hello {}! Wuz goodie my boi!".format(name))
```

Comment:
I would recommend using the f-string aproach for this as a more modern way of assembling the string.  See https://docs.python.org/3/tutorial/inputoutput.html